src/main/java/com/danielagapov/spawn/Controllers/EventController.java

```diff
@@ -123,4 +124,57 @@ public ResponseEntity<List<UserDTO>> getUsersParticipatingInEvent(@PathVariable
             return new ResponseEntity<>(HttpStatus.INTERNAL_SERVER_ERROR);
         }
     }
+
+    // full path: /api/v1/events/{eventId}/participating?userId={userid}
+    @GetMapping("events/{eventId}?userId/participating?userId={userid}")
+    public ResponseEntity<Boolean> isUserParticipating(@PathVariable UUID eventId, @RequestParam UUID userId) {
+        try {
+            if (eventService.getUserParticipationStatus(eventId, userId) == UserParticipationStatus.participating) {
+            return new ResponseEntity<>(true, HttpStatus.OK);
```

Comment:
```suggestion
                return new ResponseEntity<>(true, HttpStatus.OK);
```
Not a big deal here, but could you please make sure your IDE formats on save for the future?